test/ansible/Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM quay.io/operator-framework/ansible-operator:dev
+FROM quay.io/operator-framework/ansible-operator:v1.2.0
```

Comment:
Why not dev? Wouldn't we want to build this off of a base image that picks up changes in a PR?

Reply:
It is replaced in the shell; https://github.com/operator-framework/operator-sdk/pull/4273/files#diff-aded6410d6e48600cccc2fb400fbc86ac4fc7d2fc581385d611bc1388a90301cR51

in the follow up #4312 we will remove the shell replace as well and do that via samples stack.

Reply:
That makes sense, thanks